FxAClient/Frontend/SignIn/FxAContentViewController.swift

```diff
@@ -139,6 +139,12 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
 
     // Dispatch webkit messages originating from our child webview.
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage) {
+        // Make sure we're communicating with a trusted page. That is, ensure the origin of the
```

Comment:
I'd like you to say that this doesn't apply to WebChannels and link to https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/WebChannel.jsm.  (We should implement a WebChannel abstraction for Firefox for iOS some day, if another consumer arises.)